common/client-core/src/cli_helpers/client_list_gateways.rs

```diff
@@ -56,14 +56,16 @@ where
                 identity: remote_details.gateway_id,
                 active: active_gateway == Some(remote_details.gateway_id),
                 typ: GatewayType::Remote.to_string(),
-                endpoint: Some(remote_details.gateway_listener),
+                endpoint: Some(remote_details.gateway_listeners.primary),
```

Comment:
would it work if we clumped both primary and secondary listeners under on `endpoint` field?